File: ShadowEditor.Model/System/UserEditModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ShadowEditor.Model.System
{
    /// <summary>
    /// 用户编辑模型
    /// </summary>
    public class UserEditModel
    {
        /// <summary>
        /// 编号
        /// </summary>
        public string ID { get; set; }

        /// <summary>
        /// 用户名
        /// </summary>
        public string Username { get; set; }

        /// <summary>
        /// 密码（md5+盐）
        /// </summary>
        public string Password { get; set; }

        /// <summary>
        /// 姓名
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// 角色ID
        /// </summary>
        public string RoleID { get; set; }

        /// <summary>
        /// 性别：0-未设置，1-男，2-女
        /// </summary>
        public int Gender { get; set; }

        /// <summary>
        /// 手机号
        /// </summary>
        public string Phone { get; set; }

        /// <summary>
        /// 电子邮件
        /// </summary>
        public string Email { get; set; }

        /// <summary>
        /// QQ号
        /// </summary>
        public string QQ { get; set; }
    }
}

File: ShadowEditor.Server.Go/model/system/user_edit_model.go
package system

// UserEditModel 用户编辑模型
type UserEditModel struct {
	// 编号
	ID string
	// 用户名
	Username string
	// 密码（md5+盐）
	Password string
	// 姓名
	Name string
	// 角色ID
	RoleID string
	// 组织机构ID
	DeptID string
	// 性别：0-未设置，1-男，2-女
	Gender int
	// 手机号
	Phone string
	// 电子邮件
	Email string
	// QQ号
	QQ string
}
